doc/active_docs/account_management_api.json

```diff
@@ -9771,15 +9771,15 @@
           },
           "user_key": {
             "type": "string",
-            "description": "User Key (API Key) of the application to be created.",
+            "description": "User Key (API Key) of the application to be created.\n\n**Only alphanumeric characters `[0-9, a-z, A-Z]`, hyphen-minus (`-`), between 5 and 256 characters long. No spaces allowed.**",
```

Comment:
What about `_` and `.`? They are also valid characters.
Also, less than 5 are allowed.